server.js

```diff
@@ -15,6 +15,61 @@ app.get("/", function (request, response) {
 
 // TODO add your routes and helper functions here
 
-const listener = app.listen(process.env.PORT, function () {
+app.get("/booking", function (req, res) {
+  res.send(bookings);
+});
+
+app.get("/booking/:id", function (req, res) {
+  const bookingId = +req.params.id;
+  const booking = bookings.find((booking) => booking.id === bookingId);
+  if (!booking) {
+    res.status(404).send("404 error");
```

Comment:
Nice use of status codes here, however you could be more descriptive with the error you send back. This error will be sent to the front-end, so something like `res.status(404).json({"message": "Invalid booking id ${bookingId}"})` might be nicer